packages/desktop-client/src/components/transactions/TransactionsTable.tsx

```diff
@@ -915,13 +974,13 @@ const Transaction = memo(function Transaction({
         }),
       );
     }
-  }
+  };
 
-  function onUpdateAfterConfirm(name, value) {
+  const onUpdateAfterConfirm: TransactionUpdateFunction = (name, value) => {
     const newTransaction = { ...transaction, [name]: value };
 
     // Don't change the note to an empty string if it's null (since they are both rendered the same)
-    if (name === 'note' && value === '' && transaction.note == null) {
+    if (name === 'notes' && value === '' && transaction.notes == null) {
```

Comment:
Out of curiosity, why did the possible name change? Was this broken before?

Reply:
I'm not sure, probably a typo? It's been 'notes' and not 'note' since I started contributing.